README.md

```diff
@@ -120,11 +120,28 @@ make -j`sysctl -n hw.logicalcpu` -s
 ```
 
 #### Dependency installation notes
+
+##### psql client
 To run the `psql` client, install the `postgresql-client` package or modify `PATH` and `LD_LIBRARY_PATH` to include `pg_install/bin` and `pg_install/lib`, respectively.
 
+##### integration tests
 To run the integration tests or Python scripts (not required to use the code), install
 Python (3.9 or higher), and install python3 packages using `./scripts/pysync` (requires [poetry>=1.3](https://python-poetry.org/)) in the project directory.
 
+##### missing postgres configuration
+If you face with the following issue when you are trying to build neon
+```bash
+Configuring Postgres v14 build
+env: /Users/userA/git/neon/vendor/postgres-v14/configure: No such file or directory
+make: *** [/Users/userA/git/neon/pg_install//build/v14/config.status] Error 127
+```
+Check if you have cloned the submodules with the patched PostgreSQL. To do this, check 
+if the folder `neon/vendor/postgres-v14` contains some files there. If not, execute the 
+following command:
+```bash
+# from https://community.neon.tech/t/problem-installing-neon-locally-on-fedora/258
+git submodule update --init --recursive --depth 2 --progress .
+```
```

Comment:
Above in the README there is already a suggestion of using `git clone --recursive` for this reason (lines 94, 111).

As an alternative idea: We could detect the absense of submodules in the makefile, and fix it for the user near: https://github.com/neondatabase/neon/blob/b1a11261527c740f8abb222bbcaf8852d62b9b17/Makefile#L73-L80

Reply:
Hi @koivunej , your comment makes sense. I dropped my change in the Readme, and I added this warning in makefile.

Reply:
This is looking great! I'll resolve this soon once I've finished a test run.